components/TournamentRings.tsx

```diff
@@ -113,14 +113,14 @@ export const TournamentRings = ({
             onClick={() => toggleShowRing(ring.id)}
             style={{ marginTop: "5px" }}
           >
-            {showRing ? (
+            {showRing == ring.id ? (
               <KeyboardArrowDownRoundedIcon />
             ) : (
               <KeyboardArrowRightRoundedIcon />
             )}
             {ring.name}
           </a>
-          {!showRing ? (
+          {showRing != ring.id ? (
             <div></div>
```

Comment:
Onks tää tyhjä div kuinka elintärkeä? Ilman sitähän tän jälkimmäisen ehtolauseen vois kirjoittaa ihan vain 
```
{showRing == ring.id && 
  <div>
    ...
```

Reply:
Ja jos se on elintärkeä, niin voisko tässäkin sit kuitenkin laittaa `showRing == ring.id` -haaran ensin? Muuten näyttää vähän hämmentävältä, kun just hetkeä aikaisemmin haarat on toisin päin.